tests/serverpod_test_server/test_e2e/record_types_test.dart

```diff
@@ -231,6 +271,29 @@ void main() {
 
       expect(result, record);
     });
+    test(
+        'Given the test server, when a record with an empty Map<int, int> is send to the server, then it is returned empty.',
```

Comment:
Nit: I realise the description is this on the other test, but I think the "Given the test server" is completely useless and the wrong way to organise the test description. 

To me the given should describe the precondition / scenario we are doing and when what action we are testing.


`Given an empty map when sending a request to the server then ...`

Reply:
Updated the description.